src/main/scala/scorex/core/api/http/ApiRoute.scala

```diff
@@ -15,25 +15,32 @@ trait ApiRoute extends Directives {
   val context: ActorRefFactory
   val route: Route
 
-  implicit val timeout = Timeout(settings.timeout)
+  implicit lazy val timeout: Timeout = Timeout(settings.timeout)
```

Comment:
Is this `timeout` still used? Maybe this line could be removed?

Reply:
Or is it required as an implicit argument somewhere?

Reply:
It is required in a lot of API routes as a timeout for requests to actors. Check `ApiRouteWithFullView` for example